tests/unit/nupic/encoders/scalar_test.py

```diff
@@ -410,6 +413,36 @@ def testGetBucketInfoIntResolution(self):
     self.assertEqual(4.5, encoder.topDownCompute(encoder.encode(4.5))[0].scalar)
 
 
+  def testCapNProtoSerialization(self):
+    """Test ScalarEncoder CapNProto serialization implementation"""
+    proto1 = ScalarEncoderProto.new_message()
+    self._l.write(proto1)
```

Comment:
While this would be a useful additional test, we should have one where you encode something before serialization and then again at the end to make sure the deserialized and original get same encodings